src/main/java/nl/cherement/jak/config/WebBasicSecurity.java

```diff
@@ -0,0 +1,56 @@
+package nl.cherement.jak.config;
+
+import nl.cherement.jak.service.UserDetailsServiceImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.CorsConfigurationSource;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+
+import static nl.cherement.jak.config.SecurityConstants.SIGN_UP_URL;
+
+@Configuration
+@EnableWebSecurity
+public class WebBasicSecurity extends WebSecurityConfigurerAdapter {
+
+
+    private BCryptPasswordEncoder bCryptPasswordEncoder;
```

Comment:
Kan dit niet autowired? Omdat we ook een @Bean hebben hiervan enzo

Reply:
Opgelost